Source/Utils/WarpXUtil.cpp

```diff
@@ -214,6 +214,78 @@ WarpXParser makeParser (std::string const& parse_function, std::vector<std::stri
     return parser;
 }
 
+void CheckGriddingForRZSpectral ()
+{
+#if (defined WARPX_DIM_RZ) && (defined WARPX_USE_PSATD)
+// When using the RZ spectral solver, the Hankel transform cannot be
+// divided among multiple blocks. Each block must extend over the
+// entire radial extent.
+// The grid can be divided up along z, but the number of blocks
+// must be >= the number of processors.
+
+    int max_level;
+    Vector<int> n_cell(AMREX_SPACEDIM, -1);
+
+    ParmParse pp_amr("amr");
+
+    pp_amr.get("max_level",max_level);
+    pp_amr.getarr("n_cell",n_cell,0,AMREX_SPACEDIM);
+
+    Vector<int> blocking_factor_x(max_level+1);
+    Vector<int> max_grid_size_x(max_level+1);
+
+    // Set the radial block size to be equal to the radial grid size.
+    blocking_factor_x[0] = n_cell[0];
+    max_grid_size_x[0] = n_cell[0];
+
+    for (int lev=1 ; lev <= max_level ; lev++) {
+        // This assumes a refinement ratio of 2
+        blocking_factor_x[lev] = blocking_factor_x[lev-1]/2;
+        max_grid_size_x[lev] = max_grid_size_x[lev-1]/2;
+    }
```

Comment:
Why isn't it 
```c++
        blocking_factor_x[lev] = blocking_factor_x[lev-1]*2;
          max_grid_size_x[lev] = max_grid_size_x[lev-1]*2;
```
?

Reply:
Yes, you are correct. I've worked with multigrid Poisson solvers too much where the higher levels are coarser. :)

I'll change this to use refRatio also.